spec/models/occupation_standard_spec.rb

```diff
@@ -316,6 +316,13 @@
       expect(occupation_standard.similar_programs.pluck(:id)).to match_array [similar_program1.id, similar_program2.id]
     end
 
+    it "returns up to 5 occupations" do
+      occupation_standard = create(:occupation_standard, title: "Human Resource Specialist")
+      create_list(:occupation_standard, 10, title: "Human Resource Specialist")
```

Comment:
If we make the `5` a constant, then can you stub it to be something slightly smaller so we are creating fewer DB records in the test?